components/create_post/create_post.jsx

```diff
@@ -651,12 +705,12 @@ export default class CreatePost extends React.Component {
         }
 
         let preview = null;
-        if (this.state.fileInfos.length > 0 || this.state.uploadsInProgress.length > 0) {
+        if (this.props.draft.fileInfos.length > 0 || this.props.draft.uploadsInProgress.length > 0) {
```

Comment:
Instead of using the props in the render method, can we instead set the draft into state from the props on mount and on channel change? Similar to how create_comment.jsx works now

Otherwise we can have some issues with the cursor jumping around because drafts are getting loaded in from other tabs

Reply:
@jwilander message is set only on those two conditions above. It is not based on props directly so there shouldn't be any jumping around I think. Progress and filesInfo is based on props though.

Reply:
Ok that should be fine, we can ask @lindalumitchell to test as well

Reply:
Perfect. If there are any issues we can revisit this.